apps/client/src/features/rundown/event-block/EventBlock.utils.ts

```diff
@@ -18,24 +12,9 @@ export function formatDelay(timeStart: number, delay: number): string | undefine
   return `New start ${timeTag}`;
 }
 
-export function formatOverlap(timeStart: number, previousStart?: number, previousEnd?: number): string | undefined {
-  const noPreviousElement = previousEnd === undefined || previousStart === undefined;
-  if (noPreviousElement) return;
+export function formatOverlap(gap: MaybeNumber, isNextDay: boolean): string | undefined {
```

Comment:
suggestion, could we add some jsdocs here to explain whats going on?
```suggestion
/**
 * Creates a string representation of the overlap or gap between two events.
 * @example formatOverlap(1000, false) => "Gap 1s"
 * @example formatOverlap(-1000, false) => "Overlap 1s"
 * @example formatOverlap(1000, true) => "Gap 1s (next day)"
 */
export function formatOverlap(gap: MaybeNumber, isNextDay: boolean): string | undefined {
```